app/lib/router/shell_routers/quick_jump_shell_router.dart

```diff
@@ -0,0 +1,19 @@
+import 'package:acter/common/utils/routes.dart';
+import 'package:acter/common/widgets/dialog_page.dart';
+import 'package:acter/features/search/pages/quick_jump.dart';
+import 'package:acter/router/router.dart';
+import 'package:flutter/material.dart';
+import 'package:go_router/go_router.dart';
+
+List<RouteBase> makeQuickJumpShellRoutes(ref) {
```

Comment:
This one is especially made for desktop case, the ```search_shell_router``` is for mobile navigation. Without this, this'd lead to error .i.e. quick jump shell opens up dialog rather than page view.

Reply:
Understood but better to keep this routes in same files with appropriate naming and comments so that it is easy to get idea of this things.